models/aether_lat-lon/work/input.nml

```diff
@@ -0,0 +1,325 @@
+&probit_transform_nml
+   /
+
+&algorithm_info_nml
+   qceff_table_filename = ''
+   /
+
+&quality_control_nml
```

Comment:
add outlier_threshold = 3

Reply:
I added that, and 'input_qc_threshold   = 3.0'.  This input.nml hasn't been used directly for assimilation, so it may be missing other variables, or have suboptimal values.
@johnsonbk may also have an input.nml that's been updated to do an assimilation.